src/V8JsiRuntime.cpp

```diff
@@ -1,24 +1,22 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-#include "V8JsiRuntime_impl.h"
-
-#include "libplatform/libplatform.h"
-#include "v8.h"
-
-#include "V8Platform.h"
-#include "public/ScriptStore.h"
-
 #include <atomic>
 #include <cstdlib>
 #include <list>
 #include <mutex>
 #include <sstream>
 
+#include "V8JsiRuntime_impl.h"

```

Comment:
We typically want the main .h file to be the first one. It is also better to put all standard headers after custom header - this way we can partially ensure that custom headers have all required includes.